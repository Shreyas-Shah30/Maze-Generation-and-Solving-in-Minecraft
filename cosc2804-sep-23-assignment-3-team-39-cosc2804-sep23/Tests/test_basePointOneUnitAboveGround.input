1
2
4024 64 4226
4024 65 4226
5 5
xxxxx
....x
xxx.x
x...x
xxxxx
6
