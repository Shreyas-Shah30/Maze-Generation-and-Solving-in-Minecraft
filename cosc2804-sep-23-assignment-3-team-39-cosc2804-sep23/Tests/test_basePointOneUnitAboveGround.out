
Welcome to MineCraft MazeRunner!
--------------------------------

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- GENERATE MAZE -------------
1) Read Maze from terminal
2) Generate Random Maze
3) Back

Enter Menu item to continue: 
Enter the basePoint of the maze:
Input Error: Please enter a y-coordinate exactly one unit above the ground.
Enter the length and width of the maze:
Maze generated successfully
**Printing Maze**
BasePoint: (4024, 65, 4226)
Structure:
xxxxx
....x
xxx.x
x...x
xxxxx
**End Printing Maze**

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 
Input Error: Enter a number between 1 and 6.
Input Error: Enter a number between 1 and 6.
Input Error: Enter a number between 1 and 6.
Input Error: Enter a number between 1 and 6.
Input Error: Enter a number between 1 and 6.
The End!

